crates/scsi/src/backend.rs

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: Apache-2.0 or BSD-3-Clause
+
+use std::{
+    fs::File,
+    path::PathBuf,
+    sync::{Arc, RwLock},
+};
+
+use crate::scsi::emulation::{
```

Comment:
We are following this ordering of using modules in this crate:
- std
- other dependencies
- super/crate ones.

Move this one along with crate::vhu_scsi please.